temba/channels/models.py

```diff
@@ -2588,8 +2615,8 @@ def squash_counts(cls):
         # get the unique ids for all new ones
         start = time.time()
         squash_count = 0
-        for count in ChannelCount.objects.filter(id__gt=last_squash).order_by('channel_id', 'count_type', 'day')\
-                                                                    .distinct('channel_id', 'count_type', 'day'):
+        for count in ChannelCount.objects.filter(id__gt=last_squash).order_by('channel_id', 'count_type', 'day') \
+                .distinct('channel_id', 'count_type', 'day'):
```

Comment:
bunch of whitespace edits in this commit, why is that? these really confuse these kinds of diffs, not sure if your editor is set up differently or whatnot but try to avoid.